taworks/taform/tests.py

```diff
@@ -41,10 +41,52 @@ def test_student_form_is_valid(self):
         self.assertEqual(submitted.current_program, 'MMSC')
         self.assertEqual(submitted.citizenship, True)
 
-    def test_student_form_error_message
+    def test_student_form_error_message(TestCase):
+        """
+        Error message test to be written when form is complete with error messages
+        TODO...............
```

Comment:
i wanted that there for structure. so @amyleblond knows where to start when she finishes the error messages.